doc/api/net.md

```diff
@@ -138,7 +138,8 @@ server.listen(() => {
 });
 ```
 
-Don't call `server.address()` until the `'listening'` event has been emitted.
+If you call `server.address()` before the `'listening'` event has been
```

Comment:
To be pedantic `null` is returned also after `close()`.

Reply:
Fixed 👍